packages/debugger/app/components/frame-debugger.tsx

```diff
@@ -102,7 +102,12 @@ function FrameDebuggerFramePropertiesTableRow({
     // @todo frame here can be Partial if there are errors we should handle that somehow
     const flattenedFrame = getFrameFlattened(result.frame as Frame);
 
-    delete flattenedFrame["frames.js:version"]; // TODO: get correct frames.js version
+    // remove version info because it is populated by getFrameFlattened call but we want real value from result
+    delete flattenedFrame["frames.js:version"];
```

Comment:
should `getFrameFlattened` not be modified to use the frame definition's version if it is available?

Reply:
`frames.js:version` is not defined on `Frame` type so that's why I did this, but I can change that of course.

Reply:
what do you think of exposing a framesVersion option in getFrameFlattened?

Reply:
vs modifying the `Frame` type

Reply:
I think that could cause issues if you have frames defined by something else than `frames.js`, in that case fallback version of our package would be used but it should not be, right?

Reply:
> You mean passing version as an option to getFrameFlattened ? That could do the job.

yes. and if it's undefined we use the package.json version

Reply:
yes, but maybe it should just be an object that gets merged at the end so it acts like an overrides object

```ts
getFrameFlattened(result.frame as Frame, {
    framesVersion: result.frame['frames.js:version']
})
```

```ts
return {
...,
...overrides
}
```

Reply:
what about the cases when we debug frames that aren't rendered by `frames.js` ?

Reply:
updated, it's not coupled to frames.js because it's a generic override

Reply:
I changed the function signature as well to handle partial frames